### DISCUSSION TITLE
Git push when using Reverse Proxy Authentication?

### DISCUSSION THREAD

**Discussion by zkvvoob** [gogs/gogs#6749] [Q&A]
Hello,

I was able to successfully set up reverse proxy authentication (Authelia + Nginx) for Gogs - i.e. the two users can log in to the web interface without any issues.

However, when I try to perform a `git push` command in a terminal, I see the following message:
```
warning: redirecting to https://auth.mydomain.com/?rd=gogs.mydomain.com/user/repo.git/
fatal: https://auth.mydomain.com/?rd=gogs.mydomain.com/user/repo.git/info/refs not valid: is this a git repository?
```

So the question is, what am I doing wrong and how do we make the two functions co-exist, please?

Thank you!

**Comment by unknwon** [comment]
Based on the error message, I think your reverse proxy is redirecting _all_ requests to your SSO(?). Ideally the reverse proxy should ignore paths that contain `.git/`?
